packages/nextjs/tsconfig.build.json

```diff
@@ -8,7 +8,7 @@
     "moduleResolution": "node",
     "importHelpers": true,
     "declaration": true,
-    "declarationMap": false,
+    "declarationMap": true,
```

Comment:
❓ is this only executed only for development builds ?

Reply:
Any reason not to include the declarations for prod as well?